llvm/include/llvm/Telemetry/Telemetry.h

```diff
@@ -0,0 +1,133 @@
+//===- llvm/Telemetry/Telemetry.h - Telemetry -------------------*- C++ -*-===//
```

Comment:
Nit: coding standards have very recently changed such that the `C++` bit of this line is no longer needed.

Reply:
what does the new header look like? (I've only seen this format ....)

Reply:
https://llvm.org/docs/CodingStandards.html#file-headers

Reply:
done